### ISSUE TITLE
Possibly confusing documentation regarding app.start()

### ISSUE THREAD

**Issue by tomwhatmore** [olivernn/davis.js#79]
The first example in the intro section is

``` javascript
var app = Davis(function () {
  this.get('/welcome/:name', function (req) {
    alert("Hello " + req.params['name'])
  })
})

app.start()
```

But in the full documentation on start() it states

> Apps created using the convinience Davis() function are automatically started.

which matches my experience of using Davis.

Does this not mean that the basic example is in fact triggering the router twice? I was getting some confusing bugs with the router starting before the rest of my page was ready as I thought it would be paused until I called app.start().

On the same topic, is there any way to disable this automatic starting?

**Comment by olivernn** [comment]
Good catch, that example is out of date. Your app is automatically started when creating it with the `Davis` function. Calling start on an already started app won't do anything so it is safe to do so.

Currently there is no way to disable the automatic starting, you can of course stop the app immediately, for even more control you can instantiate an instance of `Davis.App` yourself, e.g.

```
var app = new Davis.App
app.get('/welcome/:name', function (req) { ... })
// whatever you need to do before starting the app
app.start()
```

**Comment by tomwhatmore** [comment]
Thanks, that clarifies the issue. I didn't see anything in the docs on the `new Davis.App` method of defining the routes, but I was probably looking in the wrong place. That seems to be exactly what I need.
